.bazelrc

```diff
@@ -1,6 +1,7 @@
 # Default build options
-build --copt -std=c++11
-build --copt -D_GLIBCXX_USE_CXX11_ABI=0
+build --copt -std=c++17
+build --copt -D_GLIBCXX_USE_CXX11_ABI=1
+build --experimental_repo_remote_exec
```

Comment:
From the [bazel docs](https://bazel.build/rules/lib/globals#parameters_36):

> Experimental. This parameter is experimental and may change at any time. Please do not depend on it.

Reply:
These are required in the TensorFlow bazel build at bazel 5.3.0 and TF v2.11.0

reference:

https://github.com/tensorflow/tensorflow/blob/master/.bazelrc#L438-L439

(it's defined common, which is applied to all options)